app/config.py

```diff
@@ -340,6 +339,13 @@ class Config(object):
                 'schedule': crontab(hour=4, minute=0),
                 'options': {'queue': QueueNames.PERIODIC},
             },
+            'send-scheduled-comp-and-pen-sms': {
+                'task': 'send-scheduled-comp-and-pen-sms',
+                # At every minute past every hour from 13 through 21 on every day-of-month from 21 through 28
+                # TODO test: use this as real value after testing - crontab(hour='13-21', day_of_month='21-28'),
+                'schedule': crontab(minute='0,15,30,45'),
```

Comment:
We should strive to have as few TODOs as possible. Does this value need to be changed now? Will it changed later? If we need a ticket, please create one and add it to the TODO, or convert the TODO into some helpful comment about testing and let's be sure the value is set as intended at this time, please.

Reply:
I set this to the value in the TODO. I was leaving it to execute every 15 min to complete my testing